site/source/user-manual/upgrade-hardware/ssd-upgrade.rst

```diff
@@ -0,0 +1,107 @@
+.. _ssd-upgrade:
+
+================
+Upgrade Your SSD
+================
+
+This guide will go over how to switch out an old drive on your Embassy One to a different drive and migrate the data from the old one to the new one.
+
+This will preserve the entire state of your Embassy, inlcluding all Tor and LAN addresses as well as channels you have open on the lightning network!
+
+This is not the same as backing up services and recovering them onto a new Embassy.
+
+#. Start by shutting down your Embassy:
+
+    .. figure:: /_static/images/walkthrough/shutdown.png
+        :width: 50%
+
+#. Remove the power and Ethernet cables:
+    
+    .. figure:: /_static/images/hardware-pics/back-no-cables.jpg
+        :width: 50%
+
+#. Unscrew the two screws here and remove the USB adaptor:
+
+    .. figure:: /_static/images/hardware-pics/back-screws-usb-loop.jpg
+        :width: 50%
+
+#. Remove the micro SD card from the front:
+
+    .. figure:: /_static/images/hardware-pics/front-sd-card.jpg
+        :width: 50%
+
+#. Remove the bottom four screws:
+
+    .. figure:: /_static/images/hardware-pics/bottom-screws.jpg
+        :width: 50%
+
+#. Carefully slide the contents of the case out.  You will see the SSD on the bottom of the assembly:
+
+    .. figure:: /_static/images/hardware-pics/old-drive-exposed.jpg
+        :width: 50%
+
+#. Remove the SSD by disassembling the Embassy as necessary until you can detach the SSD, then connect it to an enclosure or adaptor as shown:
```

Comment:
Need a link to somewhere that shows the construction of this - it's a bit frustrating to have so much detail and then be told "now take the whole thing apart."  There should be images in our construction guides you can just link to that section.  Same below

Reply:
Agreed, I'm going to need to embellish this section.